standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/HMSHandler.java

```diff
@@ -7431,6 +7431,13 @@ public boolean delete_column_statistics_req(DeleteColumnStatisticsRequest req) t
       if (!isPartitioned || req.isTableLevel()) {
         ret = rawStore.deleteTableColumnStatistics(parsedDbName[CAT_NAME], parsedDbName[DB_NAME], tableName, colNames, engine);
         if (ret) {
+          Map<String, String> parameters = table.getParameters();
```

Comment:
can we just remove the deleted `columns` from the COLUMN_STATS_ACCURATE? using the `StatsSetupConst.removeColumnStatsState`